src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2789,8 +2834,8 @@ public void deleteBranch(String name) throws GitException, InterruptedException
 
     }
 
-
```

Comment:
Please don't cleanup whitespace as part of a pull request.  It creates many more merge conflicts for pending pull requests (including those closed and labeled with 'Later')

Reply:
fixed